tests/conftest.py

```diff
@@ -1730,8 +1730,17 @@ def __dut_reload(duts_data, node=None, results=None):
         logger.error('Missing kwarg "node" or "results"')
         return
     logger.info("dut reload called on {}".format(node.hostname))
-    node.copy(content=json.dumps(duts_data[node.hostname]["pre_running_config"], indent=4),
+    node.copy(content=json.dumps(duts_data[node.hostname]["pre_running_config"]["host"], indent=4),
```

Comment:
instead of using "host", can we use empty string "" , we use None or empty string to refer to default namespace.

Reply:
Pushed changes to use 'None' as the key instead of 'host'